ci_framework/roles/cifmw_cephadm/tasks/cephadm_config_set.yml

```diff
@@ -47,6 +47,25 @@
     {{ cifmw_cephadm_container_ns }}/{{ cifmw_cephadm_container_image }}
   changed_when: false
 
+- name: Set haproxy container image in ceph configuration
+  when:
+    - cifmw_cephadm_haproxy_container_image is defined
+    - not (cifmw_cephadm_default_container | default(false) | bool)
+  become: true
+  ansible.builtin.command: |
```

Comment:
Just a style suggestion. If you change the yaml pipe `|` to `>-` you can get rid of `\` at the very end of each line.

Reply:
yeah, indeed... i left the block scalar style of the existing tasks for coherence. IMHO i'd like to propose a refactor of this tasks file (using a dict to define custom container images) so i will take care of style issues there if you don't mind. :)